R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
+    c("POSIXct", "POSIXt"),
+    "[^/]+",
+    lubridate::as_datetime,
+    format = "date-time",
+    location = c("query", "path"),
+    realType = "string"
+  )
+  addApiInfo(
+    "date",
+    c("Date"),
+    "[^/]+",
```

Comment:
(continuation)

(untested)
```suggestion
    # https://regex101.com/r/qH0sU7/1
    "^\\d{4}-\\d{2}-\\d{2}$",
```

Reply:
That's pretty much what I added - but without the ^ and $ and with those four slash d's -  `\\\\d`

Test added too - so resolving